src/FilesystemSyncTaskHelper.php

```diff
@@ -238,24 +295,26 @@ private function canSyncFileToDatabase(SplFileInfo $splFileInfo)
      * @param SplFileInfo $splFileInfo
      * @throws \SilverStripe\ORM\ValidationException
      */
-    private function syncFileToDatabase(SplFileInfo $splFileInfo)
+    private function syncFileToDatabase(array $splFileInfo)
```

Comment:
Can I recommend that you change the variable name? `$splFileInfo` was already a pretty bad name and now it means even less considering the type change.

Reply:
Haha, fair enough.